clients/admin-ui/src/features/config-wizard/AddSystem.tsx

```diff
@@ -0,0 +1,156 @@
+import { Box, Heading, SimpleGrid, Stack, Text } from "@fidesui/react";
+
+import { useAppDispatch, useAppSelector } from "~/app/hooks";
+import ConnectedCircle from "~/features/common/ConnectedCircle";
+import { useFeatures } from "~/features/common/features";
+import {
+  AWSLogoIcon,
+  DataFlowScannerLogo,
+  ManualSetupIcon,
+  OktaLogoIcon,
+} from "~/features/common/Icon";
+import { selectDataFlowScannerStatus } from "~/features/plus/plus.slice";
+import { ClusterHealth, ValidTargets } from "~/types/api";
+
+import { changeStep, setAddSystemsMethod } from "./config-wizard.slice";
+import SystemOption from "./SystemOption";
+import { SystemMethods } from "./types";
+
+const DataFlowScannerOption = ({ onClick }: { onClick: () => void }) => {
```

Comment:
Small nit. Maybe `DataFlowSannerOption` should be separated out into it's own file since `SystemOption`is

Reply:
good point, I'll move it into `SystemOption` so that they're all together